README.rst

```diff
@@ -124,7 +124,7 @@ Server profiles:
 Testing
 -------
 The PyKMIP test suite is composed of two parts: a unit test suite composed of
-over 600 unit tests, and an integration test suite that runs various tests
+over 750 unit tests, and an integration test suite that runs various tests
```

Comment:
That's a good point. I always update the README before a release; I'll make this change then. For now I'll simply remove this update and just leave it as is.